extensions/resteasy-reactive/quarkus-resteasy-reactive-kotlin-serialization-common/runtime/src/main/java/io/quarkus/kotlin/serialization/JsonConfiguration.java

```diff
@@ -0,0 +1,107 @@
+package io.quarkus.kotlin.serialization;
+
+import java.util.StringJoiner;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class JsonConfiguration {
```

Comment:
Ideally we have `QuarkusUnitTest` tests for each of these

Reply:
would you write tests with each configuration entry set to a different value?  that just feels like testing kotlin serialization's code than anything else.  now, i wouldn't mind testing that we're properly pulling all the values out of the config file and population the instance correctly.  I'm not sure exactly to *do* that but if you can point me in the right direction, i'll take care of that.  I'm also going to write a test to make sure that we have fields for all the possible config values so that future updates catch any new values that we're not capturing in quarkus.  also, there are other serialization schemes (protobuf, e.g.) that are nominally supported here but we're not explicitly supporting yet.

Reply:
For Jackson, we have various tests for features enabled by config - see https://github.com/quarkusio/quarkus/tree/main/extensions/jackson/deployment/src/test/java/io/quarkus/jackson/deployment.

Obviously it's not practical to have tests for everything, but maybe some for the most basic features?

Reply:
Ah!  These are the examples I've been looking for.  I'll write a handful to check the more significant config options.  Thanks.

Reply:
Great!